x-pack/solutions/security/plugins/security_solution/common/api/detection_engine/rule_management/bulk_actions/bulk_actions_route.gen.ts

```diff
@@ -131,6 +132,10 @@ Only valid when query property is undefined.
    * Gaps range end, valid only when query is provided
    */
   gaps_range_end: z.string().optional(),
+  /**
+   * Gap status to filter rules with gaps by status (used together with gaps_range_*).
+   */
+  gap_status: AggregatedGapStatus.optional(),
```

Comment:
It seems like we have two distinct definitions of `gap_status`, now:

1. Used as a filter, it means "only return gaps that have this particular status"
2. Used in a response, it means "this is the highest priority status amongst the gaps in question"

I would suggest that we don't conflate the two of these by calling them both `gap_status`, and use the more explicit `aggregated_gap_status` when applicable.